BaseClasses.py

```diff
@@ -756,19 +756,24 @@ def can_reach_region(self, spot: str, player: int) -> bool:
         return self.multiworld.get_region(spot, player).can_reach(self)
 
     def sweep_for_events(self, locations: Optional[Iterable[Location]] = None) -> None:
+        logging.error("sweep_for_events has been renamed to sweep_for_advancements. The functionality is the same. "
```

Comment:
This should probably use Utils.deprecate

Reply:
Tru